packages/manager/apps/container/tsconfig.json

```diff
@@ -20,6 +20,6 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["src"],
+  "include": ["src", "vite-env.d.ts"],
```

Comment:
I don't think this should be included, if it is, it means it will be built and we don't need it on runtime. It's only useful for the IDE and since it's a `.d.ts` file, `tsc` is able to know the types that are defined there